include/renewable_hidden_items.h

```diff
@@ -1,7 +1,7 @@
 #ifndef GUARD_RENEWABLE_HIDDEN_ITEMS_H
 #define GUARD_RENEWABLE_HIDDEN_ITEMS_H
 
-bool32 sub_815D834(void);
+bool32 ReadTrainerHillAndValidate(void);
```

Comment:
move this to cereader_tool?

Reply:
Thats the header of the C file its defined in. Oddly it seems to be sandwiched between renewable items stuff, in emerald that function is in `ereader_helpers.c`

Reply:
wdym? is the first function in that file i.e. the top of its .text section. move it to the end of cereader_tool.c

Reply:
Oh whoops you're right, skimming the file I thought `sRenewableHiddenItems` was a function